library/src/test/java/com/karumi/headerrecyclerview/HeaderRecyclerViewAdapterBaseTest.java

```diff
@@ -37,145 +36,142 @@
 public abstract class HeaderRecyclerViewAdapterBaseTest<VH extends RecyclerView.ViewHolder, H, T>
     extends RobolectricTest {
 
-  private static final int TYPE_ITEM = -1;
-  private static final int TYPE_HEADER = -2;
-  private static final ViewGroup ANY_VIEW_GROUP = mock(ViewGroup.class);
-  private static final int HEADER_POSITION = 0;
-  private static final int ITEM_POSITION = 1;
-
-  protected HeaderRecyclerViewAdapter<VH, H, T> adapter;
-
-  @Before public void setUp() {
-    adapter = getAdapterUnderTest();
-  }
+  public static final int TYPE_ITEM = -1;
+  public static final int TYPE_HEADER = -2;
+  public static final int POSITION_ZERO = 0;
+  public static final int ANY_POSITION = 3;
+  public static final int POSITION_ONE = 1;
+  public static final int HEADER_POSITION = 0;
+  public static final int ANY_NON_HEADER_POSITION = 4;
```

Comment:
You are right. I did it within the Extract>Constant of the IDE and the IDE extracts them as public.